entity-types/infra-awsbackup/dashboard.json

```diff
@@ -0,0 +1,613 @@
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT sum(aws.backup.NumberOfBackupJobsCompleted) AS 'NumberOfBackupJobsCompleted' TIMESERIES AUTO FACET aws.backup.ResourceType"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              },
+              "yAxisRight": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "NumberOfBackupJobsFailed",
+            "layout": {
+              "column": 5,
+              "row": 6,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT sum(aws.backup.NumberOfBackupJobsFailed) AS 'NumberOfBackupJobsFailed' TIMESERIES AUTO FACET aws.backup.ResourceType"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              },
+              "yAxisRight": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "NumberOfBackupJobsExpired",
+            "layout": {
+              "column": 9,
+              "row": 6,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT sum(aws.backup.NumberOfBackupJobsExpired) AS 'NumberOfBackupJobsExpired' TIMESERIES AUTO FACET aws.backup.ResourceType"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "zero": true
+              },
+              "yAxisRight": {
+                "zero": true
+              }
+            }
+          }
+        ]
+      },
+      {
```

Comment:
Hi @rahul188  the validation of the dashboard fail, please include only one page[]

```
../entity-types/infra-awsbackup/dashboard.json invalid
[
  {
    "instancePath": "/pages",
    "schemaPath": "#/properties/pages/maxItems",
    "keyword": "maxItems",
    "params": {
      "limit": 1
    },
    "message": "must NOT have more than 1 items"
  }
]
```